popupDialogExample/PopupDialogExample.js

```diff
@@ -1,47 +1,65 @@
-import React, { Component } from 'react';
-import { View, Text, StyleSheet } from 'react-native';
-import { RoundButton } from 'react-native-button-component';
+import React, {Component} from 'react';
+import {View, Text, StyleSheet, TouchableHighlight, Dimensions} from 'react-native';
 import PopupDialog from 'react-native-popup-dialog';
 
+const {width, height} = Dimensions.get('window');
```

Comment:
I think if the width & height use uppercase will be better.

FYI: have a sugar sytax for ES7, `const {width: WIDTH, height: HEIGHT} = Dimensions.get('window');`